### ISSUE TITLE
train loss and valid loss are nan

### ISSUE THREAD

**Issue by 674106399** [fastai/fastai#941]
I trained 2 models(resnet101 and densenet161) on the same dataset, which is a multi-label dataset

learner = ConvLearner(dataset, torchvision.models.densenet161, metrics=[accuracy_thresh, f2_opt], 
                      ps=0.4, callback_fns=ShowGraph, pretrained=False, loss_fn=nn.BCEWithLogitsLoss())
learner = learner.to_fp16()

but during the training, both show that:

epoch train loss valid loss accuracy_thresh f2_opt
0 nan nan 0.997788 0.000000

71.39% [6930/9707 3:56:27<1:34:45 nan]

it is strange that both of the 2 models' print-out information is: 0 nan nan 0.997788 0.000000

on the beginning, the loss display correctly, but about 1 hour later it turned into nan
who can tell me what's wrong with it?

**Comment by sgugger** [comment]
It sounds like you are training with a learning rate too high. In any case, this isn't an issue with the library so please use the [forum](https://forums.fast.ai/).
